src/styles/02-molecules/edition-card/_edition-card.scss

```diff
@@ -1,67 +1,63 @@
 .edition-card {
   @include space-stack-s;
 
-  border: 1px solid $gray-light;
   border-radius: 5px;
-  padding: $space-xs $space $space $space;
+  border: 1px solid $gray-light;
+  display: flex;
+  flex-flow: column wrap;
+  padding: $space $space-l;
+
+  @include breakpoint($medium) {
+    justify-content: space-between;
+    flex-flow: row wrap;
+  }
+
+  &__heading {
+    flex: 100%;
+  }
 
   &__card-content {
+    align-items: center;
     display: flex;
-    flex-flow: column nowrap;
-    align-items: flex;
-    justify-content: center;
+    flex-flow: column wrap;
 
     @include breakpoint($medium) {
       flex-flow: row nowrap;
-      justify-content: flex-start;
+      flex: 0 1 400px;
```

Comment:
![Screen Shot 2020-05-08 at 2 00 38 PM](https://user-images.githubusercontent.com/3579472/81434641-7c6bfc00-9134-11ea-9892-f15d71763332.png)

Is this intended to truncate at 400px?

Reply:
It was meant as an initial value... for the above, we should set it to grow as well as shrink with a `1`.